packages/network-controller/tests/provider-api-tests/block-hash-in-response.ts

```diff
@@ -0,0 +1,230 @@
+import {
+  ProviderType,
+  waitForNextBlockTracker,
+  withMockedCommunications,
+  withNetworkClient,
+} from './helpers';
+
+type TestsForRpcMethodThatCheckForBlockHashInResponseOptions = {
+  providerType: ProviderType;
+  numberOfParameters: number;
+};
+
+export const testsForRpcMethodsThatCheckForBlockHashInResponse = (
```

Comment:
Future PR: We should use `function` here not only to make them stand out but also to enforce that these functions are JSDoc'ed properly.